pkg/jobrunaggregator/jobrunhistoricaldataanalyzer/analyzer.go

```diff
@@ -142,31 +142,42 @@ func (o *JobRunHistoricalDataAnalyzerOptions) compareAndUpdate(newData, currentD
 		}
 
 		newP99 := getDurationFromString(new.GetP99())
-		newP95 := getDurationFromString(new.GetP99())
+		newP95 := getDurationFromString(new.GetP95())
 		d := parsedJobData{}
 
 		// If the current data contains the new data, check and record the time diff
 		if old, ok := currentData[key]; ok {
+			oldP95 := getDurationFromString(old.GetP95())
 			oldP99 := getDurationFromString(old.GetP99())
 
 			d.HistoricalData = new
 			d.DurationP99 = newP99
 			d.DurationP95 = newP95
 			d.JobResults = new.GetJobRuns()
 
-			timeDiff := newP99 - oldP99
-			percentDiff := 0.0
+			timeDiffP95 := newP95 - oldP95
+			timeDiffP99 := newP99 - oldP99
+			percentDiffP95 := 0.0
+			if oldP95 != 0 {
+				percentDiffP95 = (float64(timeDiffP95) / float64(oldP95)) * 100
+			}
+			percentDiffP99 := 0.0
 			if oldP99 != 0 {
-				percentDiff = (float64(timeDiff) / float64(oldP99)) * 100
+				percentDiffP99 = (float64(timeDiffP99) / float64(oldP99)) * 100
+			}
+			if newP95 > oldP95 && percentDiffP95 > o.leeway {
+				increaseCountP95 += 1
+				d.TimeDiffP95 = timeDiffP95
+				d.PercentTimeDiffP95 = percentDiffP95
+				d.PrevP95 = oldP95
 			}
-			if newP99 > oldP99 && percentDiff > o.leeway {
-				increaseCount += 1
-				d.TimeDiff = timeDiff
-				d.PercentTimeDiff = percentDiff
+			if newP99 > oldP99 && percentDiffP99 > o.leeway {
+				d.TimeDiffP99 = timeDiffP99
+				d.PercentTimeDiffP99 = percentDiffP99
 				d.PrevP99 = oldP99
 			}
-			if newP99 < oldP99 {
-				decreaseCount += 1
+			if newP95 < oldP95 {
+				decreaseCountP95 += 1
```

Comment:
P99 is what we're actually enforcing, addition of the P95 is just to help us see if there's something that needs manual investigation. 

The decrease count probably should stay with P99, as that seems like it might be what we'd expect when viewing the file.